bin/render-activity-stream-html.js

```diff
@@ -29,6 +29,7 @@ const DEFAULT_OPTIONS = {
  */
 function templateHTML(options, html) {
   const isPrerendered = !!html;
+  const scriptForPrerendering = `\n<script src="${options.baseUrl}data/content/activity-stream-initial-state.js"></script>`;
```

Comment:
nit: It'd be even more readable if you just embedded a new line in the string rather than using /n.